testing/src/network/topology.rs

```diff
@@ -56,7 +56,9 @@ async fn line(nodes: &mut Vec<Node<LedgerStorage>>) {
     // Start each node with the previous as a bootnode.
     for node in nodes {
         if let Some(addr) = prev_node {
-            node.config.bootnodes.write().push(addr);
+            let mut bootnodes = (&**node.config.bootnodes.load()).clone();
+            bootnodes.push(addr);
+            node.config.bootnodes.store(Arc::new(bootnodes));
```

Comment:
ok, so I guess this would be that one use we have for runtime modification of bootnodes